cheri_compressed_cap_common.h

```diff
@@ -83,6 +92,8 @@ struct _cc_N(cap) {
     uint8_t cr_flags;    /* Flags */
     uint8_t cr_reserved; /* Remaining hardware-reserved bits to preserve */
     uint8_t cr_tag;      /* Tag */
+    uint8_t cr_bounds_valid /* Set if bounds decode was given an invalid cap */;
+    uint32_t pad4;
```

Comment:
Compiler should add this anyway?

Reply:
The last one probably yes. Because this was interacting with TCG (which goes outside the type system), I just wanted to get really explicit on sizing the structure correctly. Plus, this way I can see what fields I have left if I want to steal some extra space.

Reply:
Use offsetof?

Reply:
I was about to say that its more helpful for the other pad fields. I now notice they are gone because I stole them for pesbt and cr_bounds_valid. Can delete this last one if people think its cleaner. There is a static assert in QEMU to catch the case the size changes unexpectedly.